cmplibrary/src/main/java/com/sourcepoint/cmplibrary/consent/ClientEventManager.kt

```diff
@@ -61,7 +61,7 @@ private class ClientEventManagerImpl(
                 ActionType.CUSTOM,
                 ActionType.MSG_CANCEL,
                 ActionType.PM_DISMISS -> {
-                    if (!action.requestFromPm || action.singleShotPM) {
+                    if (!action.requestFromPm) {
```

Comment:
I think we should move away from using `!` and use `.not()` instead, since it is more of a kotlinish way